packages/dapp/src/components/WelcomeWrapper/WelcomeWrapper.style.ts

```diff
@@ -0,0 +1,66 @@
+import { Box, Breakpoint, Typography, styled } from '@mui/material';
+
+export const Background = styled('div')(({ theme }) => ({
+  backgroundColor: 'transparent',
+  position: 'fixed',
+  width: '100%',
+  top: '50%',
+  bottom: 0,
+  zIndex: 1300,
+  height: 'calc( 50% + 64px )',
+  [`@media screen and (max-height: 490px)`]: {
+    height: 'auto',
+    top: 0,
+  },
+}));
+
+export const CustomColor = styled(Typography)(({ theme }) => ({
+  background:
+    theme.palette.mode === 'dark'
+      ? 'linear-gradient(270deg, #D35CFF 0%, #BEA0EB 94.17%)'
+      : 'linear-gradient(270deg, #31007A 0%, #8700B8 94.17%);',
```

Comment:
Why do we have such a value of `94.17`?

Reply:
Those values are copied from the Figma layout: It´s the gradient on that Welcome-Title.
![image](https://github.com/lifinance/jumper.exchange/assets/43956540/30edc145-ed2b-402e-a9e4-c1e70b0b2910)